lib/config/pricing/src/index.js

```diff
@@ -148,7 +151,7 @@ const id = (oid, rtype, ppid, time, auth, cb) => {
   if(!rtype) {
     debug('Undefined resource type');
     return cb(undefined,
-      businessError(true, 'Pricing config - Undefined resource type'))
+      businessError('Pricing config - Undefined resource type'))
```

Comment:
Interesting, so maybe there are two different configurations here:

- A flow with 3 steps, step 1 succeeds, step 2 fails, step 3 fails too because step 2 failed to obtain the required data for it. For this case, I'd suggest to stop the execution of the flow in step 2. The reported error is simply the error that caused step 2 to fail.

- Nested calls, with service 1 calling service 2 calling service 3, service 3 fails and reports an error, causing service 2 to fail and report its own error, causing service 1 to fail and report its own error as well. In this case we could either propagate and report the error from service 3 as is, or report the nested errors like I was trying to describe in my previous comment, looking like a sort of 'business error stack trace'. I'm not sure yet about useful that'd be so it's probably better to keep it simple for now and simply propagate the original error from service 3 as is.

What do you think?